### ISSUE TITLE
`{@inheritDoc}` does not count as type documentation anymore

### ISSUE THREAD

**Issue by derrabus** [slevomat/coding-standard#1601]
Starting with 8.13.2, I see a lot of false positives regarding doc blocks popping up.

```
FILE: src/MyClass.php
------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
 12 | ERROR | Method \App\MyClass::returnSomething() does not have return type hint nor @return annotation for its return value.
    |       | (SlevomatCodingStandard.TypeHints.ReturnTypeHint.MissingAnyTypeHint)
------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
```

src/MyClass.php:

```php
<?php

namespace App;

class MyClass implements MyInterface
{
    /**
     * Implementation of {@see MyInterface::returnSomething()}.
     *
     * {@inheritDoc}
     */
    public function returnSomething()
    {
        return null;
    }
}
```

phpcs.xml

```xml
<?xml version="1.0"?>
<ruleset
        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
        xsi:noNamespaceSchemaLocation="vendor/squizlabs/php_codesniffer/phpcs.xsd"
>
    <arg name="basepath" value="."/>
    <arg name="extensions" value="php"/>
    <arg name="parallel" value="80"/>
    <arg name="colors"/>

    <arg value="ps"/>

    <file>src</file>

    <rule ref="SlevomatCodingStandard.TypeHints.ReturnTypeHint">
        <exclude name="SlevomatCodingStandard.TypeHints.ReturnTypeHint.MissingNativeTypeHint"/>
    </rule>
</ruleset>
```

Until 8.13.1, the `{@inheritDoc}` block was enough to document the return type.

**Comment by kukulich** [comment]
Thank you for your report.

Fixed in https://github.com/slevomat/coding-standard/commit/4b2af2fb17773656d02fbfb5d18024ebd19fe322

**Comment by derrabus** [comment]
Thank you for your quick response. With the new 8.13.4 release, some of those false positives are fixed, but not all of them. I can try to isolate a better reproducer later. Alternatively, you can checkout the 3.6.x branch of https://github.com/doctrine/dbal/.
